zerver/openapi/zulip.yaml

```diff
@@ -1168,6 +1168,10 @@ paths:
                                 the new stream exists (for private streams, only subscribers and
                                 organization administrators will receive this event).
 
+                                This event is also sent when the user gains access to a previously
+                                inaccessible stream either due to user being subscribed to a private
+                                stream or if a private stream is made public.
```

Comment:
The code for sending creation events on user gaining stream access, was not added with a feature level, so not added "Changes" entry here.